### PR TITLE
Add labels to example services

### PR DESCRIPTION
**Pull request by galexrt**
See #157.

***

Should an example ServiceMonitor for Prometheus be added too?

### CODE REVIEW

File: example/alertmanager-service.yaml [hunk 1]
```diff
@@ -2,6 +2,9 @@ apiVersion: v1
 kind: Service
 metadata:
   name: alertmanager-main
+  labels:
+    app: alertmanager
+    instance: main
```
Comment (fabxc) [inline example/alertmanager-service.yaml]: I think in our stateful sets etc. we went with `app: alertmanager` and `alertmanager: main`.
`instance` is only really meaningful in combination of both labels whereas the other one can serve as a meaningful selector in itself.

Comment (galexrt) [inline example/alertmanager-service.yaml]: The same for the Promtetheus? Like this `app: prometheus` and `prometheus: main`?

Comment (brancz) [inline example/alertmanager-service.yaml]: Yep

**Comment by galexrt** [comment]
Renamed the label names.

**Comment by brancz** [comment]
lgtm

**Pull request merged**